src/main/java/frc/robot/Robot.java

```diff
@@ -28,6 +28,7 @@ public void robotPeriodic() {
 
   @Override
   public void disabledInit() {
+    robotContainer.shooterHoodSubsystem.setPercentage(0);
```

Comment:
Does this work, can you move actuators in `disabledInit()`? I honestly don't know.

Reply:
It runs before teleop and auto I believe, I was using it to reset the actuator on startup. We'll want to decide on a proper reset location once we have it mounted.